frontend/packages/ceph-storage-plugin/src/constants/common.ts

```diff
@@ -2,6 +2,7 @@ export const CEPH_HEALTHY = 'is healthy';
 export const CEPH_DEGRADED = 'health is degraded';
 export const CEPH_ERROR = 'health is in error state';
 export const CEPH_UNKNOWN = 'is not available';
+export const CEPH_BRAND_NAME = 'Red Hat Ceph Storage';
```

Comment:
`export const cephBrandName = (t: TFunction) => t('ceph-storage-plugin~Red Hat Ceph Storage');`
also, `import { TFunction } from 'i18next';` on the top.